Andela-Ray/PropertyPro-lite
# #167268244 Fix UI templates - remove service type

**What does this PR do?**
Removes the service type option from the UI

**Description of Task to be completed?**
Remove service type option from:
-add_property.html
-edit_property.html
-view_property.html
-property.html

**How should this be manually tested?**
Visit the gh-page of the templates at 'https://andela-ray.github.io/PropertyPro-lite/index.html' and navigate to the relevant pages to observe the fixed UI views

**What are the relevant pivotal tracker stories?**
#167268244
